proposals/dao/tip_121c_pt2.ts

```diff
@@ -0,0 +1,112 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  PcvStats,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+
+/*
+
+Tribe Redemption 
+
+*/
+
+// Circulating amount of TRIBE, which is redeemable for underlying PCV assets
+// TODO: Update with final circulating TRIBE figure
+const REDEEM_BASE = ethers.constants.WeiPerEther.mul(458_964_340);
+
+// Lido deposit balance, being withdrawn and sent to Tribe Redeemer
+const STETH_DEPOSIT_BALANCE = '50296523674661485703301';
```

Comment:
On block number update, this number must change (rebases every 24h)

Reply:
The proposal moves 100% of the balance, so the proposal validate can be updates with an .at.least(...) instead of .equal(...)

Reply:
Yep exactly, is an .at.least(). It's a lower bound